### PR TITLE
Added changes to ignore blank space inputs

### PR DESCRIPTION
**Pull request by pavan-pan**
modified:   analysis.c
modified:   main.c
modified:   requests.c
When spaces or new line is entered, no action is taken. Later can be improved by displaying some error messages

### CODE REVIEW

**Comment by ritwik12** [comment]
@pavan-pan Thanks for your PR. Please change the title of the PR to make it more appropriate.

File: analysis.c [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 //request analysis
 
+
```
Comment (ritwik12) [inline analysis.c]: @pavan-pan This change is by mistake?

**Comment by ritwik12** [comment]
@pavan-pan I have checked your PR and it is working well. Thanks for your contribution.

File: analysis.c (hunk 1 above)
Comment (ritwik12) [reply analysis.c]: @pavan-pan Ping

Comment (pavan-pan) [reply analysis.c]: yup. seems so. i should be careful the next time. Thanks for pointing it out!

Comment (ritwik12) [reply analysis.c]: Correct it please, I have to merge this now.

Comment (pavan-pan) [reply analysis.c]: I have changed it

**Comment by ritwik12** [comment]
@pavan-pan Thanks!!

**Comment by ritwik12** [comment]
@all-contributors please add @pavan-pan for code

**Pull request merged**